packages/react-core/src/filters/applyFiltersToTileContent.js

```diff
@@ -0,0 +1,82 @@
+import { buildBinaryFeatureFilter } from './Filter';
+
+export function applyFiltersToTileContent(tileContent, filters) {
+  if (!filters || !Object.keys(filters).length || !tileContent) {
+    return tileContent;
+  }
+
+  const filterFn = buildBinaryFeatureFilter({ filters });
+
+  const applyFiltersFn = (idx, data) =>
+    (!data.attributes || data.attributes.getFilterValue.value[idx]) &&
+    filterFn(idx, data);
+
+  const pointsGetFilterValue = tileContent.points.featureIds.value.map((_, idx) =>
+    applyFiltersFn(idx, tileContent.points)
+  );
+
+  const polygonsGetFilterValue = applyToPolygonsLinesData(
+    tileContent.polygons,
+    applyFiltersFn
+  );
+
+  const linesGetFilterValue = applyToPolygonsLinesData(tileContent.lines, applyFiltersFn);
+
+  return {
+    ...tileContent,
+    points: {
+      ...tileContent.pois,
+      attributes: {
+        getFilterValue: {
+          value: pointsGetFilterValue,
+          size: 1
+        }
+      }
+    },
+    polygons: {
+      ...tileContent.polygons,
+      attributes: {
+        getFilterValue: {
+          value: polygonsGetFilterValue,
+          size: 1
+        }
+      }
+    },
+    lines: {
+      ...tileContent.lines,
+      attributes: {
+        getFilterValue: {
+          value: linesGetFilterValue,
+          size: 1
+        }
+      }
+    }
+  };
+}
+
+// Aux
+function applyToPolygonsLinesData(data, applyFiltersFn) {
+  const featuresGetFilterValue = new Uint16Array(data.featureIds.value.length);
```

Comment:
According to documentation https://github.com/visgl/loaders.gl/blob/master/modules/gis/docs/api-reference/geojson-to-binary.md `featureIds` can be Uint16Array | Uint32Array so we could change this by `Uint32Array`